source/preferences/manage-your-desktop-notifications.rst

```diff
@@ -0,0 +1,92 @@
+Manage your desktop notifications
+=================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |dot-badge| image:: ../images/dot-badge.png
+  :alt: A dot on the badge means you have unread activity in at least one channel you're a member of.
+  :width: 50px
+
+.. |numbered-badge| image:: ../images/numbered-badge.png
+  :alt: A numbered badge means you have at least 1 unread message, @mention, or one of your keywords has triggered a notification.
+  :width: 50px
+
+Enable notifications
+--------------------
+
+From Mattermost v9.9, Mattermost prompts you to enable notifications in the desktop app.
+
+.. image:: ../images/enable-notifications.png
+  :alt: From Mattermost v9.9, you're prompted to enable notifications.
+
+- When you select **Enable notifications**, you won't be asked again. You'll start receiving notifications for all Mattermost activity with `badges <#badge-based-notifications>`__, `banner alerts <#banner-alerts>`__ and `sounds <#notification-sounds>`__. See the section below on `customizing your notifications <#customize-your-notifications>`__ based on how you prefer to be notified about Mattermost activity in the desktop app.
```

Comment:
> - When you select **Enable notifications**, you won't be asked again.

We could include here where they can change this in the Edit Server dialog in case they need to.